ci/export-logs.js

```diff
@@ -0,0 +1,48 @@
+const AWS = require('aws-sdk')
+
+const fs = require('fs')
+const path = require('path')
+
+AWS.config.update({ 
+  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
+  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
+  region: process.env.AWS_REGION
+})
+
+s3 = new AWS.S3({ apiVersion: '2006-03-01' })
+
+const directoryPath = path.join(__dirname, '../root/.ceramic/logs')
+
+function main() {
+  fs.readdir(directoryPath, (err, files) => {
```

Comment:
@valmack for future reference, there is `fs/promises` module that exposes Promise-based API https://nodejs.org/docs/latest-v14.x/api/fs.html#fs_fs_promises_api